src/hardware/ipxserver.cpp

```diff
@@ -20,26 +20,29 @@
 
 #if C_IPX
 
+#include <atomic>
+#include <thread>
+
 #include "ipx.h"
 #include "ipxserver.h"
 #include "timer.h"
-#include <cstdlib>
-#include <cstring>
 
-constexpr int UDP_UNICAST = -1; // SDLNet magic number
+static constexpr int UDP_UNICAST = -1; // SDLNet magic number
+
+static IPaddress ipxServerIp;     // IPAddress for server's listening port
+static UDPsocket ipxServerSocket; // Listening server socket
+static SDLNet_SocketSet socket_set = nullptr;
```

Comment:
@kklobe minor: We try to eradicate inline comments on sight 😎

Reply:
hah! will try to remember that from now on.